### DISCUSSION TITLE
Support for Drag & Drop

### DISCUSSION THREAD

**Discussion by Solido** [btwld/mix#441] [Ideas]
Is there a roadmap to consult? I would add Drap&Drop with the same expressivity than PressableBox.
Thanks

**Comment by Solido** [comment]
I've seen some stuff about dragged in the code but nothing in the doc.
Must be a work in progress!

**Comment by tilucasoli** [comment]
Hi @Solido, we have implemented the state in MixWidgetState. However, we currently lack a specific widget, like Draggable, to manage it. I will add it to our roadmap. For now, you can manually enable and disable it using `MixWidgetStateController`.
